heartbeat/tests/system/test_monitor.py

```diff
@@ -41,34 +42,39 @@ def test_http(self, status_code):
         self.assert_fields_are_documented(output[0])
 
     @parameterized.expand([
-        ("8185", "up"),
-        ("8186", "down"),
+        (lambda server: "localhost:{}".format(server.server_port), "up"),
+        # This IP is reserved in IPv4
+        (lambda server: "207.254.52.220:1233", "down"),
```

Comment:
I cannot find what is this IP reserved for :thinking: https://en.wikipedia.org/wiki/Reserved_IP_addresses

Reply:
Should it be a `203.0.113.0/24`?

Reply:
Yes, I use `203.0.113.1` elsewhere, no idea where I got that weird IP from. Good find!